tests/test_views.py

```diff
@@ -0,0 +1,98 @@
+"""
+Tests for the learning assistant views.
+"""
+import json
+import sys
+from unittest.mock import MagicMock, patch
+
+from django.urls import reverse
+
+from learning_assistant.models import CoursePrompt
+from tests import LoggedInTestCase
+
+
+class CourseChatViewTests(LoggedInTestCase):
+    """
+    Test for the CourseChatView
+    """
+    sys.modules['lms.djangoapps.courseware.access'] = MagicMock()
```

Comment:
When I put these mocks in the setUp function, the modules weren't being mocked properly.

Reply:
Why do we need to mock these? Are you able to let the imports fail and then mock out the imported functions? Or does that not work if the import statement fails?

Reply:
I need to mock them so that an `ImportError` isn't thrown when running the tests. If I try to mock the imported functions, that also fails because the module isn't available. 

I tried to use `patch.dict` to update `sys.modules`, but because the tests are hitting an endpoint and not testing a specific function that can be imported, that method of patching doesn't work.